pkg/services/featuremgmt/registry.go

```diff
@@ -904,5 +904,12 @@ var (
 			FrontendOnly: false,
 			Owner:        grafanaDatabasesFrontend,
 		},
+		{
+			Name:            "managedPluginsInstall",
+			Description:     "Install managed plugins directly from plugins catalog",
+			Stage:           FeatureStageExperimental,
+			RequiresDevMode: true,
```

Comment:
Nit. Is this true?

Reply:
I wanted to be safe and test it before going to production.
Will this make my installation tests more difficult to perform? (I don't know much about our environments)

Reply:
Changed anyway XD

Reply:
Would require app_mode=development I think which is not enabled in cloud. Merging this the feature toggle will be disabled unless configured to be enabled

Reply:
Tks for pointing this out!